jobs/base.bull.job.js

```diff
@@ -11,6 +11,10 @@ class BaseJob {
 		this.logger = log.createLogger({ group: this.constructor.name, jobName: this.constructor.name, queueName });
 		this.queueName = queueName;
 		this.concurrency = concurrency;
+		const lockDuration = overrideQueueOptions && overrideQueueOptions.settings
+			? overrideQueueOptions.settings.lockDuration
+			: 60000;
+		const lockRenewTime = overrideQueueOptions && overrideQueueOptions.settings && overrideQueueOptions.settings.lockRenewTime;
```

Comment:
also seems curious to me to take values from the overrides, set defaults with them, and then merge the two objects anyways. does the `merge` function not already do that?

Reply:
Good point, this could be shortened a bit by not getting `lockRenewTime` and only setting to calculate based on `lockDuration` as default